### PR TITLE
lib: it would not clean the MODEL_URL directory

### PR DESCRIPTION
**Pull request by jaypume**
1. train in a temp directory, instead of cleaning the MODEL_URL.
2. remove some unused logs.

### CODE REVIEW

**Comment by llhuii** [comment]
can you insert prefix `lib: ` to commit to be more clear

File: lib/sedna/incremental_learning/incremental_learning.py [hunk 1]
```diff
@@ -51,17 +53,23 @@ def train(model, train_data, epochs, batch_size, class_names, input_shape,
...
+    pb_model_url_with_prefix = os.path.join(il_config.model_url,
+                                            il_config.saved_model_name)
 
-    clean_folder(il_config.model_url)
     model.train(train_data, [])  # validation data is empty.
     tf.reset_default_graph()
-    model.save_model_pb(il_config.saved_model_name)
+    LOG.info(f'starting to save model to {TRAINING_TMP_DIR}, '
+             f'model name is {il_config.saved_model_name}')
+    model.save_model_pb(TRAINING_TMP_DIR, il_config.saved_model_name)
```
Comment (llhuii) [inline lib/sedna/incremental_learning/incremental_learning.py:63]: here add a extra arg to save_model_pb in incremental_learning. I can't find the same operation in federated_learning.

Comment (jaypume) [reply lib/sedna/incremental_learning/incremental_learning.py:63]: the interfaces exposed by lib are not unified, Therefore, federated_learning is not affected. 
But interfaces unification is still a problem, we can propose another issue for tracking.

**Pull request closed without merging**